src/notation/internal/notationactioncontroller.h

```diff
@@ -230,6 +230,8 @@ class NotationActionController : public actions::Actionable, public async::Async
 
     void registerPadNoteAction(const mu::actions::ActionCode&, Pad padding);
     void registerTabPadNoteAction(const mu::actions::ActionCode&, Pad padding);
+    void notifyAboutActionIsTriggered(const mu::actions::ActionCode&);
```

Comment:
let's rename this method to `notifyAccessibilityAboutActionTriggered(const mu::actions::ActionCode& actionCode)`

Also please move these two methods after last registerAction method with adding empty line before